codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -234,7 +234,8 @@ void removeStream(DefaultStream stream) {
     /**
      * Simple stream implementation. Streams can be compared to each other by priority.
      */
-    private class DefaultStream implements Http2Stream {
+    private class DefaultStream implements Http2Stream, DefaultHttp2RemoteFlowController.FastPropertyAccess,
```

Comment:
> I don't think we should be tying the implementations of DefaultStream to DefaultHttp2RemoteFlowController. That basically means that we've lost the pluggability of the flow controller.

Uhm why? If a stream implementation other than default is used, I have a fallback for the map?